### DISCUSSION TITLE
Bot does not start on VPS hosting

### DISCUSSION THREAD

**Discussion by Syberox** [jagrosh/MusicBot#1096] [Troubleshooting] [answered]
![Безымянный](https://user-images.githubusercontent.com/74526859/148371125-caed56fc-db41-4066-a1bf-c1d95353751e.png)

java -Dnogui=true -jar bot.jar

[13:37:30] [WARN] [Settings]: Failed to load server settings (this is normal if no settings have been set yet): java.nio.file.NoSuchFileException: serversettings.json
[13:37:30] [INFO] [Startup]: Loaded config from /root/bot/config.txt

After these lines, the bot is not online, it cannot start. The token entered is correct

**Comment by bosk20** [comment]
Confirm, problem is with java 17. Just downgrade to java 16 or wait for update

**Reply by Sanduhr32** [reply]
the problem is, that you are running java 17 on a single core/thread. try adding `-Djava.util.concurrent.ForkJoinPool.common.parallelism=1` after `java`

**Comment by Sanduhr32** [comment, marked as answer]
If you run Java 17 on single core machines you currently have to manually enable `-Djava.util.concurrent.ForkJoinPool.common.parallelism=1`
